includes/ajax-functions.php

```diff
@@ -437,14 +437,42 @@ function edd_ajax_get_states_field() {
 function edd_ajax_download_search() {
 	global $wpdb;
 
-	$search  = esc_sql( sanitize_text_field( $_GET['s'] ) );
+	$search   = esc_sql( sanitize_text_field( $_GET['s'] ) );
+	$excludes = ( isset( $_GET['post'] ) ? (array) $_GET['post'] : array() );
```

Comment:
@ghost1227, @pippinsplugins is correct, `post` here needs to change to `current_id`, that's what's being sent in the JavaScript above: https://github.com/easydigitaldownloads/Easy-Digital-Downloads/pull/3008/files#diff-4e07975920906985a5fbbe3afd720629R1046

This won't ever occur on bundles, as they are excluded in the dropdown via the html element, but in the event that it's on a download page that's not a bundle and we do have a product search...this would prevent it.